tide_site.info.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
   - dpc-sdp:tide_media
   - entity_browser:entity_browser
   - entity_browser:entity_browser_entity_form
+  - create_menus_permission:create_menus_permission
```

Comment:
@anthony-malkoun should we make this "Create menus permission" a required dependency?

Alternative would be to make optional, allowing sites to use more fine-grained alternatives or nothing at all. You could show something in the Drupal Status Report as a recommendation if the module was not installed. 

Making it a required dependency for now. If an issue does arise within the user-base, we could make it optional.

Reply:
Required dependency is fine here I think.